src/Features/GlobalProperties/PropertiesTable/PropertiesTable.tsx

```diff
@@ -213,32 +213,7 @@ function PropertiesTable({ properties }: { properties: Property[] }) {
             />
           </>
         ) : (
-          <>
-            <DataTable
-              rows={newProperties}
-              headers={headers}
-              render={({ headers }: { headers: any }) => (
-                <TableContainer>
-                  <Table>
-                    <TableHead>
-                      <TableRow className={styles.tableHeadRow}>
-                        {headers.map((header: any) => (
-                          <TableHeader
-                            key={header.key}
-                            id={header.key}
-                            className={`${styles.tableHeadHeader} ${styles[header.key]}`}
-                          >
-                            {header.header}
-                          </TableHeader>
-                        ))}
-                      </TableRow>
-                    </TableHead>
-                  </Table>
-                </TableContainer>
-              )}
-            />
-            <Error404 header={null} title="No properties to be found" message={null} />
-          </>
+          <EmptyState title="Looks like there's nothing here" message="No results from that search, try again." />
```

Comment:
Related to the previous comment. This looks like it works fine without the `header` being explicitly set to null. Makes sense bc I don't there is a default for the ErrorPage like there is the 404.